test/fixtures/shared/variables.tf

```diff
@@ -14,29 +14,31 @@
  * limitations under the License.
  */
 
+variable "project_id" {
+  description = "The project ID"
+  type        = string
+}
+
 variable "parent_id" {
   description = "The parent of this AccessPolicy in the Cloud Resource Hierarchy. As of now, only organization are accepted as parent."
+  type        = string
 }
 
 variable "policy_name" {
   description = "The policy's name."
+  type        = string
 }
 
 variable "protected_project_ids" {
   description = "Project id and number of the project within the regular service perimeter"
-  type        = "map"
```

Comment:
Types should be clarified based on Terraform 0.12 features, not removed.

Reply:
@aaron-lane, thank you for review! 
[Resolved](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls/compare/bd6511ad2b5be1ff99f7a375811b644a26590922..a6cab6903015d4eecd47c690fb3eba0798e0b659#diff-963d4c8c343b85de0fee346ecdad2ff9)